packages/contractkit/test/driver.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Address } from '../src/base'
+import { newKit } from '../src/kit'
+
+test('Test driver', async () => {
+  // Integration
+  // const kit = newKit('http://35.247.98.50:8545')
+  const kit = newKit('http://35.247.74.27:8545')
```

Comment:
Why not read the addresses the wallet kit does?
https://github.com/celo-org/celo-monorepo/blob/master/packages/walletkit/test/utils.ts#L78

Reply:
a user might want to connect to it's own node...